### PR TITLE
Css four column layout

### PR DESCRIPTION
**Pull request by james-aguirre**
Screenshot 2023-04-05 at 1 30 36 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.header {
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.column-fourth-two > img {
+  width: 20%;
+  padding-left: 130px;
```
Comment (shawnkost) [inline css-four-column-layout/styles.css:27]: You shouldn't need to be setting such high `px` values. If you need to do something this large, then you are probably doing something "hacky" to get this to work.

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.column-4 {
+  width: 100%;
+  justify-content: center;
+  padding-left: 110px;
+}
```
Comment (shawnkost) [inline css-four-column-layout/styles.css:34]: If these are supposed to be columns of 4, why are you setting the `width` to 100%? Shouldn't this be `flex-basis: 25%`?

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+.column-4-2 {
+  width: 100%;
+  justify-content: center;
+  padding-left: 70px;
+}
```
Comment (shawnkost) [inline css-four-column-layout/styles.css:40]: Same note as above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here James!

**Comment by james-aguirre** [comment]
Screenshot 2023-04-17 at 3 36 21 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**